homeassistant/components/netatmo/climate.py

```diff
@@ -252,25 +256,20 @@ def hvac_action(self) -> Optional[str]:
 
     def set_hvac_mode(self, hvac_mode: str) -> None:
         """Set new target hvac mode."""
-        mode = None
-
         if hvac_mode == HVAC_MODE_OFF:
-            mode = STATE_NETATMO_OFF
+            self.turn_off()
         elif hvac_mode == HVAC_MODE_AUTO:
-            mode = PRESET_SCHEDULE
+            if self.hvac_mode == STATE_OFF:
```

Comment:
We should use the hvac mode constants:
https://github.com/home-assistant/core/blob/58ae117bb415e949a65d61abe3b1a19bca9ebca0/homeassistant/components/climate/const.py#L4